packages/alea-frontend/pages/sys-admin.tsx

```diff
@@ -74,8 +74,16 @@ const SysAdmin: NextPage = () => {
   async function handleCreateClick() {
     if (!aclId || !resourceId || !actionId) return;
     try {
+      const isAclValid = await isValid(aclId);
+      if(!isAclValid){
+        setAclId('');
+        setError('invalid acl id');
+        setIsSubmitting(false);
+        return;
+      }
       setIsSubmitting(true);
-      await createResourceAction({ aclId, resourceId, actionId });
+      const upperCaseActionId = actionId.toUpperCase();
```

Comment:
Instead of making it upper case, use a drop-down. The user should only be able to select one of the many value of the `Action`  enum.